src/status_im/components/icons/vector_icons.cljs

```diff
@@ -3,83 +3,70 @@
-            :icons/address             (slurp-svg "./resources/icons/address.svg")
-            :icons/arrow-left          (slurp-svg "./resources/icons/arrow_left.svg")
-            :icons/arrow-right         (slurp-svg "./resources/icons/arrow_right.svg")
-            :icons/flash-active        (slurp-svg "./resources/icons/flash_active.svg")
-            :icons/flash-inactive      (slurp-svg "./resources/icons/flash_inactive.svg")
-            :icons/attach              (slurp-svg "./resources/icons/attach.svg")
-            :icons/back                (slurp-svg "./resources/icons/back.svg")
-            :icons/browse              (slurp-svg "./resources/icons/browse.svg")
-            :icons/close               (slurp-svg "./resources/icons/close.svg")
-            :icons/copy-from           (slurp-svg "./resources/icons/copy_from.svg")
-            :icons/dots-horizontal     (slurp-svg "./resources/icons/dots_horizontal.svg")
-            :icons/dots-vertical       (slurp-svg "./resources/icons/dots_vertical.svg")
-            :icons/exclamation_mark    (slurp-svg "./resources/icons/exclamation_mark.svg")
-            :icons/filter              (slurp-svg "./resources/icons/filter.svg")
-            :icons/forward             (slurp-svg "./resources/icons/forward.svg")
-            :icons/fullscreen          (slurp-svg "./resources/icons/fullscreen.svg")
-            :icons/group-big           (slurp-svg "./resources/icons/group_big.svg")
-            :icons/group-chat          (slurp-svg "./resources/icons/group_chat.svg")
-            :icons/hamburger           (slurp-svg "./resources/icons/hamburger.svg")
-            :icons/hidden              (slurp-svg "./resources/icons/hidden.svg")
-            :icons/mic                 (slurp-svg "./resources/icons/mic.svg")
-            :icons/ok                  (slurp-svg "./resources/icons/ok.svg")
-            :icons/public              (slurp-svg "./resources/icons/public.svg")
-            :icons/public-chat         (slurp-svg "./resources/icons/public_chat.svg")
-            :icons/qr                  (slurp-svg "./resources/icons/QR.svg")
-            :icons/search              (slurp-svg "./resources/icons/search.svg")
-            :icons/smile               (slurp-svg "./resources/icons/smile.svg")
-            :icons/commands-list       (slurp-svg "./resources/icons/commands_list.svg")
-            :icons/dropdown-up         (slurp-svg "./resources/icons/dropdown_up.svg")
-            :icons/dropdown            (slurp-svg "./resources/icons/dropdown.svg")
-            :icons/grab                (slurp-svg "./resources/icons/grab.svg")
-            :icons/share               (slurp-svg "./resources/icons/share.svg")
-            :icons/tooltip-triangle    (slurp-svg "./resources/icons/tooltip-triangle.svg")
-            :icons/network             (slurp-svg "./resources/icons/network.svg")})
+(def icons {:icons/chats               (js/require "./resources/icons/bottom/chats_gray.svg")
```

Comment:
@vkjr since `src/status_im/components/icons/vector_icons.cljs` is compiled both for mobile and desktop, does it make sense to try to make the code cross-platforms compatible? `[status-im.utils.platform :as platform]` should help as I understand. Does above make sense? Thanks ( P.S. If currently platform is defined as iOS, but we compile for desktop, maybe as temp solution you can just replace iOS with desktop to test the code? I'm investigating how to define desktop as separate platform for cljs project also )

Reply:
@MaxRis, in status-react desktop with electron colleagues maintain separate set of files for desktop. I didn't investigate yet how their build works but I suppose that at some moment before build they replace files from original status-react with modified. That approach can be cleaner then adding if's to all places with differences.
I suggest to leave this file as is until we decide what approach we will use. What do you think?

Reply:
@vkjr sounds good, thank you!